src/prefect/orion/schemas/core.py

```diff
@@ -30,6 +30,8 @@
     "flow_run_state_message",
 ]
 
+DEFAULT_BLOCK_SCHEMA_VERSION = "non-versioned"
```

Comment:
is there a benefit to having a default value here instead of null?

Reply:
I think it simplifies the code a little, but that's a personal opinion. We need a way to reference blocks schemas that don't have a version in the future `block_schemas/versions/{version}` and `block_schemas/versions` endpoints and a default string seemed like the simplest way to do that.